sublime-package.json

```diff
@@ -309,8 +309,40 @@
               "markdownDescription": "Add an additional delay when the auto-complete widget is currently visible. Just like `\"diagnostics_delay_ms\"`, the unit is milliseconds. The total amount of delay would be `diagnostics_delay_ms + diagnostics_additional_delay_auto_complete_ms`."
             },
             "diagnostics_highlight_style": {
-              "enum": [],
-              "deprecationMessage": "Use \"show_diagnostics_highlights\" instead",
+              "oneOf": [
+                {
+                  "enum": ["box", "underline", "stippled", "squiggly", ""],
+                },
+                {
+                  "type": "object",
+                  "properties": {
+                    "error": {
+                      "enum": ["box", "underline", "stippled", "squiggly", ""],
+                      "default": "squiggly"
+                    },
+                    "warning": {
+                      "enum": ["box", "underline", "stippled", "squiggly", ""],
+                      "default": "squiggly"
+                    },
+                    "info": {
+                      "enum": ["box", "underline", "stippled", "squiggly", ""],
+                      "default": "stippled"
+                    },
+                    "hint": {
+                      "enum": ["box", "underline", "stippled", "squiggly", ""],
+                      "default": "stippled"
+                    },
+                  },
+                  "additionalProperties": false
+                }
+              ],
+              "default": {
+                "error": "squiggly",
+                "warning": "squiggly",
+                "info": "stippled",
+                "hint": "stippled"
+              },
+              "markdownDescription": "Highlight style of code diagnostics.\nCan be a string, or a mapping of string->string for severity-based styling.\nValid string values are `\"box\"`, `\"underline\"`, `\"stippled\"`, `\"squiggly\"` or the empty string (`\"\"`).\nWhen set to a valid string value, all severities will use that style.\nWhen set to the empty string (`\"\"`), no document highlighting is requested.\nDiagnostics which span multiple lines are always drawn with the `box` style (See `\"show_multiline_diagnostics_highlights\"`)."
```

Comment:
This mentions "document highlighting is requested", but we're dealing with diagnostics, not document highlights.

Reply:
Yup, my error, will fix.